lib/markdown.js

```diff
@@ -95,7 +105,11 @@ module.exports.processQuestion = function (html) {
     const decodedContents = originalContents.replace(escapeRegex, (match, prefix, hashes) => {
       return `${prefix}${'#'.repeat(hashes.length - 1)}>`;
     });
-    const res = processor.processSync(decodedContents);
+    const res = questionProcessor.processSync(decodedContents);
     return res.contents;
   });
 };
+
+module.exports.processContent = function (original) {
+  return regularProcessor.processSync(original).contents;
```

Comment:
Can we use an async processing function here instead? I'm unsure of exactly what might be async, but in general we try to avoid anything "sync" when an async version is available. (I acknowledge that I used `processSync` above - that was a mistake 😓)

Reply:
Done.